<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerWindow</class>
 <widget class="QMainWindow" name="ViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vision Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QFrame" name="statusFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="coordsLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Coords</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="valueLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Pixel value</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="zoomLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Zoom</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="elapsedTime">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Elapsed time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Viewer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="ImageViewerWithSelector" name="graphicsView">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>153</red>
                <green>153</green>
                <blue>153</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>153</red>
                <green>153</green>
                <blue>153</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="renderHints">
           <set>QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="previewTab">
       <attribute name="title">
        <string>Preview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="ImageViewerWithSelector" name="preview">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>153</red>
                <green>153</green>
                <blue>153</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>153</red>
                <green>153</green>
                <blue>153</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="renderHints">
           <set>QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout_us">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_us"/>
    <addaction name="actionLog_2"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionHistogram"/>
    <addaction name="actionCanny"/>
    <addaction name="actionGaussianBlur"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
   <addaction name="menuAbout_us"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom100"/>
   <addaction name="actionZoomFit"/>
   <addaction name="separator"/>
   <addaction name="actionPolygonalSelection"/>
   <addaction name="actionRectSelection"/>
   <addaction name="actionPointSelection"/>
   <addaction name="actionLockSelections"/>
  </widget>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open image or video file</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save image file</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About VisionViewer</string>
   </property>
  </action>
  <action name="actionSplit_and_Merge">
   <property name="text">
    <string>Split and Merge</string>
   </property>
  </action>
  <action name="actionSave_polygon_layer">
   <property name="text">
    <string>Save polygon layer</string>
   </property>
  </action>
  <action name="actionLoad_polygon_layer">
   <property name="text">
    <string>Load polygon layer</string>
   </property>
  </action>
  <action name="actionLearnd_objects">
   <property name="text">
    <string>Learnd objects</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionZoom100">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/Zoom100.ico</normaloff>:/icon/Resources/Zoom100.ico</iconset>
   </property>
   <property name="text">
    <string>zoom100</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/ZoomFit.ico</normaloff>:/icon/Resources/ZoomFit.ico</iconset>
   </property>
   <property name="text">
    <string>zoomFit</string>
   </property>
  </action>
  <action name="actionPolygonalSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/polygon.ico</normaloff>:/icon/Resources/polygon.ico</iconset>
   </property>
   <property name="text">
    <string>polygonalSelection</string>
   </property>
  </action>
  <action name="actionRectSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/rectangle.ico</normaloff>:/icon/Resources/rectangle.ico</iconset>
   </property>
   <property name="text">
    <string>rectSelection</string>
   </property>
  </action>
  <action name="actionPointSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/points.ico</normaloff>:/icon/Resources/points.ico</iconset>
   </property>
   <property name="text">
    <string>pointSelection</string>
   </property>
  </action>
  <action name="actionLockSelections">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/Resources/openLock.ico</normaloff>:/icon/Resources/openLock.ico</iconset>
   </property>
   <property name="text">
    <string>lockSelections</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionSimilarity">
   <property name="text">
    <string>Similarity</string>
   </property>
  </action>
  <action name="actionBorders">
   <property name="text">
    <string>Borders</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionGaussianBlur">
   <property name="text">
    <string>GaussianBlur</string>
   </property>
  </action>
  <action name="actionLog_2">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageViewerWithSelector</class>
   <extends>QGraphicsView</extends>
   <header>ImageViewerWithSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
